### PR TITLE
fix: save new location and new state during schedule update (issue 159)

### PR DESCRIPTION
**Pull request by stephanepilla**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

The location and state are updatable in the scheduleCreationPopup but are actually not saved in the updateSchedule function. 

Fix for #159 : 
- Use schedule.location instead of schedule.raw.location in scheduleCreationPopup (as defined in the model) to be consistent with the detail popup
- Save sched.location and sched.state in updateSchedule function

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Comment by seonim-ryu** [comment]
Can one of the admins verify this patch?

File: src/js/controller/base.js [hunk 1]
```diff
@@ -189,6 +189,14 @@ Base.prototype.updateSchedule = function(schedule, options) {
         schedule.set('isFocused', options.isFocused);
     }
 
+    if (options.location) {
+        schedule.set('location', options.location);
+    }
+
+    if (options.state) {
```
Comment (dongsik-yoo) [inline src/js/controller/base.js]: Is this necessary for your issue? I agree this is a bug, but not related with `location`. What do you think?

Comment (stephanepilla) [reply src/js/controller/base.js]: It's another not related property, but it was also not saved. So I added it in the PR.
Yes, I would need both for my requirements.

Comment (dongsik-yoo) [reply src/js/controller/base.js]: Ok. Then I will add new property and description at `schedule.js`. It's just an information for you.
```js
/**
 * The model of calendar schedules.
 * @constructor
 * @mixes dirty
 * @mixes model
 */
function Schedule() {
...
    /**
     * location
     * @type {string}
     */
    this.location = '';

    /**
     * state
     * @type {string}
     */
    this.state = '';
...
}
```

**Comment by stephanepilla** [comment]
Good thanks.
Do you know already when you plan to release the next bugfix release ?

**Comment by dongsik-yoo** [comment]
@stephanepilla I just released [v1.5.0](https://github.com/nhnent/tui.calendar/releases/tag/v1.5.0) including this issue.

**Pull request merged**